### ISSUE TITLE
Updating the jail uninstalls jellyfin

### ISSUE THREAD

**Issue by ryannathans** [spz2k9/iocage-plugin-jellyfin#5]
I'm not sure how else to put this, but if I update the jail through the truenas gui (update button on the jail or plugin page), then all the packages get uninstalled (no jellyfin, no ffmpeg, etc)

Rolling back to a snapshot created moments before the backup restores jellyfin to the previous working state

**Comment by spz2k9** [comment]
I think it's normal for now. The v2 plugin had a install script inside and wasn't meant to be updated with the Update button of the UI. The v3 plugin which has been merge last week should fix this issue, but it won't be pushed before the next Truenas Core Update.

**Comment by spz2k9** [comment]
I've removed the command in the script as asked here : 

While IX side gets merged you can use this command to add the missing parts : 

pkg install -y ffmpeg jellyfin libva-intel-media-driver

**Comment by spz2k9** [comment]
Oups, forgot to add the link : https://github.com/ix-plugin-hub/iocage-plugin-index/pull/368

**Comment by spz2k9** [comment]
I made a temporary patch yesterday so the installer works.

**Comment by spz2k9** [comment]
Hi, is it resolved? If not you can try the manual install : 

https://github.com/spz2k9/iocage-plugin-jellyfin

**Comment by ryannathans** [comment]
I'm on version 10.8.10_1, 10.8.10_2 is shown as available, but updating through the UI is still broken. When the update finishes, the version of installed plugin becomes N/A and the problem reoccurs. 

> I made a temporary patch yesterday so the installer works.

I assume this means installing 10.8.10_2 from scratch and not upgrading to it?

Keen for v3 if it resolves everything without extra fiddling, there is no time pressure as Jellyfin server has not updated since plugin release (yet)

**Comment by spz2k9** [comment]
Yes, updating will require at least _3... but the thing is... _3 is based on 13.1 which has gone obsolete and ffmpeg doesn't work anymore... so only _4 will really fix everything. I have no eta on when it'll be merged but you could create a new jail with the manual install (which use _4) and copy / move the jellyfin configuration files / directory to the new jail. That way there wouldn't have anything to reconfigure.

I haven't tried it but the config are probably /var/lib/jellyfin and /etc/jellyfin (if it's the same as debian).

**Comment by ryannathans** [comment]
Yeah I think the whole obsoleting of 13.1 really breaks a lot

Other plugin jails all breaking too when I try to update them because stuck on 13.1 and binaries get downloaded for 13.2

We need truenas to be a bit more responsive with these updates, they are already solved problems

**Comment by foreverlz1111** [comment]
> Yeah I think the whole obsoleting of 13.1 really breaks a lot
> 
> Other plugin jails all breaking too when I try to update them because stuck on 13.1 and binaries get downloaded for 13.2
> 
> We need truenas to be a bit more responsive with these updates, they are already solved problems

The problem happened in 13.1-RELEASE, it is caused by **ffmpeg** malfunction. 

13.1-RELEASE is the base system of TrueNAS13.0-U5, I had tried many command lines, there are no ways to upgrade the exist jellyfin jail into 13.2-RELEASE, unless we manually create a empty 13.2 jail, ffmpeg works perfect in 13.2-RELEASE.

**Comment by spz2k9** [comment]
> > Yeah I think the whole obsoleting of 13.1 really breaks a lot
> > Other plugin jails all breaking too when I try to update them because stuck on 13.1 and binaries get downloaded for 13.2
> > We need truenas to be a bit more responsive with these updates, they are already solved problems
> 
> The problem happened in 13.1-RELEASE, it is caused by **ffmpeg** malfunction.
> 
> 13.1-RELEASE is the base system of TrueNAS13.0-U5, I had tried many command lines, there are no ways to upgrade the exist jellyfin jail into 13.2-RELEASE, unless we manually create a empty 13.2 jail, ffmpeg works perfect in 13.2-RELEASE.

Yeah the whole situation doesn't seem really well handled. Anyway i'm no expert, but the only way I see it could be done would be to use the manual commands to create a new 13.2 plugin for JellyFin and manually copy the config files from the 13.1 jail to the 13.2... but I don't know the location of those config files.
